rust-runtime/aws-smithy-http/src/lib.rs

```diff
@@ -21,5 +32,8 @@ pub mod retry;
 #[cfg(feature = "event-stream")]
 pub mod event_stream;
 
+#[cfg(feature = "rt-tokio")]
```

Comment:
I don't think this is right? All of ByteStream works fine without rt-tokio (except the from_path) parts

Reply:
You're right, I'll remove this